### ISSUE TITLE
Inventory list from different FCs

### ISSUE THREAD

**Issue by anndix** [CPIGroup/phpAmazonMWS#182]
Is it possible to use the library to pull report - Daily Inventory Report ? It allows to see different Skus inventory in different FCs. We have inventory in 14 FCs so would want to keep track.

**Comment by Peardian** [comment]
Yes, this library can request and retrieve reports. Check the classes with names that begin with "AmazonReport" for info on how to use them. You will need to know the name of the report you want to fetch, which you can find listed on the official website: http://docs.developer.amazonservices.com/en_US/reports/Reports_ReportType.html
